packages/perspective-viewer-datagrid/src/less/toolbar.less

```diff
@@ -87,6 +87,8 @@
 
 .button:hover {
     opacity: 1;
+    display: flex;
+    align-items: center;
```

Comment:
What's the context for this?  The children of the elements this class is applied are the same height as the container so this should be inert - and indeed testing this locally against FF+Chrome+Safari and I see no difference?

Reply:
This was to fix the alignment of the text/selectors in these buttons. Visiting https://perspective.finos.org/ on Firefox I'm able to reproduce this, but I just tested Chrome and Safari and I don't see the issue there

<img width="1007" alt="theme selector alignment" src="https://user-images.githubusercontent.com/12093492/159033201-a44163e2-3ee8-4c09-ae96-e9df085b25b3.png">
<img width="1006" alt="scroll lock alignment" src="https://user-images.githubusercontent.com/12093492/159033209-a2c6f61e-4263-46e7-9b9b-0d5500ad6822.png">

Reply:
I am seeing the same as @kevinhinterlong on FF with MacOS.

Reply:
Ok, I do see this on FF OSX 78.9, thanks for the clarification